src/App.scss

```diff
@@ -1,8 +1,49 @@
+/* eslint-disable max-len */
+
```

Comment:
Fix eslint errors, not just ignore them)

Reply:
fixed)